qdrant_client/async_qdrant_remote.py

```
@@ -2455,3 +2472,44 @@ async def collection_cluster_info(self, collection_name: str) -> types.Collectio
         ).result
         assert collection_info is not None, "Collection cluster info returned None"
         return collection_info
+
+    async def get_optimizations(
+        self, collection_name: str, completed_limit: int | None = None, **kwargs: Any
```

Ruff (0.15.1)

[warning] Unused method argument: `kwargs`

(ARG002)